api/src/main/java/jakarta/persistence/PersistenceConfiguration.java

```diff
@@ -516,6 +518,27 @@ public ValidationMode validationMode() {
         return validationMode;
     }
 
+    /**
+     * Specify the {@linkplain FetchType#DEFAULT default fetch type}
+     * for one-to-one and many-to-one associations.
+     * @param defaultFetchType the default fetch type
+     * @return this configuration
+     * @since 4.0
+     */
+    public PersistenceConfiguration defaultFetchType(FetchType defaultFetchType) {
```

Comment:
Disallow setting a default of DEFAULT.
```suggestion
     * for one-to-one and many-to-one associations. The given fetch type
     * value must be {@link FetchType#EAGER} or {@link FetchType#LAZY}.
     * @param defaultFetchType the default fetch type
     * @return this configuration
     * @since 4.0
     */
    public PersistenceConfiguration defaultFetchType(FetchType defaultFetchType) {
        if (defaultFetchType == FetchType.DEFAULT)
            throw new IllegalArgumentException("defaultFetchType: " + FetchType.DEFAULT);
```